brewtils/decorators.py

```diff
@@ -105,81 +104,79 @@ def echo_json(self, message):
     Args:
         _wrapped: The function to decorate. This is handled as a positional argument and
             shouldn't be explicitly set.
+        description: The command description. If not given the first line of the method
+            docstring will be used.
+        parameters: A list of Command parameters. It's recommended to use @parameter
+            decorators to declare Parameters instead of declaring them here, but it is
+            allowed. Any Parameters given here will be merged with Parameters sourced
+            from decorators and inferred from the method signature.
```

Comment:
This is interesting. So instead of having `@parameter`, the plugin developer could have a list of them like `paramters=[Parameter(...), Parameter(...)]`